recipes/cgns/build.sh

```diff
@@ -0,0 +1,33 @@
+#! /usr/bin/env bash
+
+_build_shared=ON
+if [[ `uname -s` == 'Darwin' ]]; then
+    export MACOSX_DEPLOYMENT_TARGET=""
```

Comment:
it seems like you should not reset this

Reply:
v3.4.0 would be nice, and built with hdf5 1.10. The difference in terms of build between 3.3 and 3.4 should be minimal.

Thanks.

Reply:
Fixed with 7f4f8f8.